### DISCUSSION TITLE
Giving this a shot

### DISCUSSION THREAD

**Discussion by almarklein** [vispy/vispy2-sandbox#1] [General]
Hi @rougier and @rossant,

This is my first time using the discussions-feature of GH :) Let's see how this works out. I also created a project (kanban-like board) as an experiment to see if this works for collecting use-cases.

**Comment by djhoese** [comment]
* Problem 1: "vipsy2poc" -> "vispy2poc"? (s and p are swapped)
* Question 1: Do we need a totally different repository? If we plan on merging this to main vispy eventually then the git histories won't combine.

**Reply by rougier** [reply]
And what the "2poc" means ?

**Reply by rossant** [reply]
I suspect "Vispy 2 proof of concept" ;)

**Reply by almarklein** [reply]
oh dear, I've managed to screw up the repo name. Yes I meant "vispy2 proof of concepot.

**Reply by almarklein** [reply]
done

**Reply by almarklein** [reply]
> Do we need a totally different repository?

This is intended for discussions, experiments and proof of concepts. All the stuff that we *do not* want in the main repo.

**Comment by rougier** [comment]
Why not simply vispy2 then?

**Reply by almarklein** [reply]
I guess because (so far) this is intended for *prepping* vispy2, right?

**Comment by rougier** [comment]
Ok. poc sounds a bit weird to me but I can live with it.

**Reply by djhoese** [reply]
POC is a difficult acronym to use. Given the world we live in today POC (People of Color) is maybe not the best acronym to use. `vispy2-sandbox`? `vispy2-test`? `vispy2-playground`?

**Reply by almarklein** [reply]
Fair enough. vispy2-sandbox it is.
